django/contrib/postgres/fields/hstore.py

```diff
@@ -13,9 +13,9 @@
 
 class HStoreField(Field):
     empty_strings_allowed = False
-    description = _('Map of strings to strings')
+    description = _('Map of strings to strings or nulls')
     default_error_messages = {
-        'not_a_string': _('The value of "%(key)s" is not a string.'),
+        'not_a_string_or_none': _('The value of "%(key)s" is not a string or None.'),
```

Comment:
I don't know if changing the key is a good idea since would be backwards-incompatible.